js/packages/core/lib/signals.ts

```diff
@@ -85,3 +85,21 @@ export function select(g: ElemNode, a: ElemNode, b: ElemNode): NodeRepr_t {
 export function hann(t: ElemNode): NodeRepr_t {
   return el.mul(0.5, el.sub(1, el.cos(el.mul(2.0 * Math.PI, t))));
 }
+
+/**
+ * Converts a MIDI note number to a frequency in Hz.
+ * 
+ * Middle C4 is MIDI note 60, which corresponds to 261.63 Hz.
+ */
+export function midi2frequency(midi: ElemNode): NodeRepr_t {
```

Comment:
Super nitpick, but I'd prefer `midi2freq` and `freq2midi` here just for brevity. When composing larger graphs I find short (but understandable) names to be helpful